api/HIDConnectionEvent.json

```diff
@@ -0,0 +1,152 @@
+{
+  "api": {
+    "HIDConnectionEvent": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/HIDConnectionEvent",
+        "spec_url": "https://wicg.github.io/webhid/#dom-hidconnectionevent",
+        "support": {
+          "chrome": {
+            "version_added": "89"
+          },
+          "chrome_android": {
+            "version_added": false
+          },
+          "edge": {
+            "version_added": "89"
+          },
+          "firefox": {
+            "version_added": false
+          },
+          "firefox_android": {
+            "version_added": false
+          },
+          "ie": {
+            "version_added": false
+          },
+          "opera": {
+            "version_added": "75"
+          },
+          "opera_android": {
+            "version_added": false
+          },
+          "safari": {
+            "version_added": false
+          },
+          "safari_ios": {
+            "version_added": false
+          },
+          "samsunginternet_android": {
+            "version_added": false
+          },
+          "webview_android": {
+            "version_added": false
+          }
+        },
+        "status": {
+          "experimental": false,
```

Comment:
Per https://github.com/mdn/browser-compat-data/blob/main/docs/data-guidelines.md#choosing-an-experimental-status the experimental bits should be set to true.

Reply:
I've sent https://github.com/mdn/browser-compat-data/pull/11069 to make the existing entries match.